bindings/python/crocoddyl/core/activations/2norm-barrier.cpp

```diff
@@ -0,0 +1,46 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2021, LAAS-CNRS, Airbus
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/core/core.hpp"
+#include "python/crocoddyl/core/activation-base.hpp"
+#include "crocoddyl/core/activations/2norm-barrier.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeActivation2NormBarrier() {
+  
+  bp::class_<ActivationModel2NormBarrier, bp::bases<ActivationModelAbstract> >(
+      "ActivationModel2NormBarrier",
+      "An 2-norm activation model for a defined barrier alpha\n\n"
+      "If the residual is over an alpha threshold, this function imposes a quadratic term. \n"
+      "In short, the activation value is 0 if the residual is major to alpha, otherwise, it is \n"
+      "equals to 0.5 *(||r|| - alpha)^2",
+      bp::init<std::size_t, double>(bp::args("self", "nr", "alpha"),
```

Comment:
You need to make the alpha argument as an optional one (to reflect the same API of the c++ code). The code will be
```cpp
       bp::init<std::size_t, bp::optional<double> >(bp::args("self", "nr", "alpha"),
```

and then you need to describe this in the documentation as
```cpp
                     ":param alpha: activation threshold (default 0.1)"))
```